srcpkgs/CKAN/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'CKAN'
+pkgname=CKAN
+version=1.30.4
+revision=1
+archs="x86_64* i686*"
```

Comment:
why no arm?

Reply:
It makes sense only to support archs that are supported by KSP itself - thus no ARM

Reply:
done